server/src/main/java/org/elasticsearch/action/support/UnsafePlainActionFuture.java

```diff
@@ -39,7 +48,6 @@ public UnsafePlainActionFuture(String unsafeExecutor, String unsafeExecutor2) {
     boolean allowedExecutors(Thread thread1, Thread thread2) {
```

Comment:
How about a rename (here and in the superclass)?

```suggestion
    boolean allowedExecutors(Thread blockedThread, Thread completingThread) {
```

Reply:
Makes sense, done in: 247d7c7e89e9dde6429ceacd45d174c17d4037bd